### PR TITLE
chore(execution-engine): Some stream-related LambdaError are unjoinab…

### PR DESCRIPTION
**Pull request by raftedproc**
…le b/c: canon stream replaces normal stream, when canon stream  is used, it is materialized and its size is known

### CODE REVIEW

**Review by mikevoronov — APPROVED** [review]
utACK.

Let's delete these two tests; they no longer test anything.

File: crates/air-lib/trace-handler/src/data_keeper/trace_slider.rs [hunk 1]
```diff
@@ -83,6 +83,7 @@ impl TraceSlider {
     }
 
     pub(crate) fn set_subtrace_len(&mut self, subtrace_len: usize) -> KeeperResult<()> {
+        println!("set_subtrace_len subtrace_len: {}", subtrace_len);
```
Comment (mikevoronov) [inline crates/air-lib/trace-handler/src/data_keeper/trace_slider.rs:86]: don't forget to delete it before merging

File: crates/air-lib/test-utils/src/lib.rs [hunk 2]
```diff
@@ -166,5 +166,12 @@ pub fn is_interpreter_succeded(result: &RawAVMOutcome) -> bool {
 }
 
 pub fn check_error(result: &RawAVMOutcome, error: impl ToErrorCode + ToString) -> bool {
+    println!(
```
Comment (mikevoronov) [inline crates/air-lib/test-utils/src/lib.rs:169]: don't forget to delete it before merging

File: crates/air-lib/trace-handler/src/data_keeper/trace_slider.rs (hunk 1 above)
Comment (raftedproc) [reply crates/air-lib/trace-handler/src/data_keeper/trace_slider.rs]: Those lines accedently slipped into the patch. Thx for your vigilance.

**Pull request merged**